ios/Surveys/Sources/Presentation/Modules/SurveyQuestions/SurveyQuestionsView.swift

```diff
@@ -0,0 +1,141 @@
+//  swiftlint:disable function_body_length
+//  SurveyQuestionsView.swift
+//  Surveys
+//
+//  Created by Su Ho on 27/12/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import SwiftUI
+import SwiftUIPager
+
+struct SurveyQuestionsView: View {
+
+    @ObservedObject private var viewModel: SurveyQuestionsViewModel
+    @EnvironmentObject private var navigator: Navigator
+    @StateObject private var page: Page = .first()
+
+    var body: some View {
+        switch viewModel.state {
+        case .idle:
+            content().onAppear { viewModel.loadData() }
+        case .loading:
+            content(isLoading: true)
+        case .loaded:
+            content()
+        case let .failure(message):
+            content().defaultAlert(message: message)
+        }
+    }
+
+    var background: some View {
+        ZStack {
+            GeometryReader { proxy in
+                URLImage.urlString(viewModel.imageURLString)
+                    .resizable()
+                    .scaledToFill()
+                    .frame(width: proxy.size.width, height: proxy.size.height)
+                    .clipped()
+                    .blur(radius: 20.0)
+            }
+            .ignoresSafeArea()
+
+            Rectangle()
+                .foregroundColor(.clear)
+                .background(
+                    LinearGradient(colors: [.clear, .black], startPoint: .top, endPoint: .bottom)
+                )
+                .opacity(0.6)
+                .ignoresSafeArea()
+        }
+    }
+
+    init(viewModel: SurveyQuestionsViewModel) {
+        self.viewModel = viewModel
+    }
+
+    private func content(isLoading: Bool = false) -> some View {
+        ZStack {
+            background
+            if isLoading {
+                ProgressView().progressViewStyle(CircularProgressViewStyle(tint: .white))
+            } else {
+                Spacer(minLength: 16.0)
+                Pager(
+                    page: page,
+                    data: viewModel.uiModel?.questions ?? [],
+                    id: \.id,
+                    content: { question in
+                        ScrollView {
+                            VStack(alignment: .leading) {
+                                Text(question.progress)
+                                    .font(Typography.neuzeitSLTStdBookHeavy.font(size: 15))
```

Comment:
```suggestion
                                    .font(Typography.neuzeitSLTStdBookHeavy.font(size: 15.0))
```

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/129/commits/a1e271726322e4efcb4e720191bc6fa7aa9c6462